platform/remarkable/disable-wifi.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/sh
 
-# disable wifi and remove modules
+# clean stop of wpa_supplicant service, used by xochitl
+if systemctl is-active -q wpa_supplicant; then
+    systemctl stop wpa_supplicant
+fi
 
-# clean stop (if it's running) of main wpa_supplicant service, used by xochitl
-systemctl stop wpa_supplicant
-# clean stop of non-service wpa_supplicant, if running
-wpa_cli terminate 2>/dev/null
+# clean stop of non-service wpa_supplicant
+if pidof wpa_supplicant >/dev/null; then
```

Comment:
Completely personal nit: I tend to prefer `pkill -0 process_name` (or `kill -0 pid` if the pid is known) for "is this thing alive?" checks ;).

Reply:
Ah, that looks cleaner. Unfortunately, the rM does not have pkill natively, and the pid is not known beforehand. Do you recommend any other alternatives?

Reply:
Ah, that's an unexpected hurdle ;).

Probably not then, as `kill -0 $(pidof wpa_supplicant)` isn't really any better (and will bew screwy if there are multiple matches).

Reply:
(I would have mentioned pgrep, at least for some similar use-cases (mainly when you want to match on the commandline rather than the process name), but, IIRC, pkill *is* pgrep, so I assume it's not available either anyway ^^)

Reply:
That's right. I compiled the `procps-ng` binaries for rM to test your suggestions and they work quite nicely. Will use it in my own scripts. Thanks for the insight!